shared/chat/inbox/row/divider.js

```diff
@@ -46,7 +46,7 @@ const Divider = ({isExpanded, isBadged, toggle}: DividerProps) => (
   >
     <DividerBox>
       <Box style={_dividerIconStyle}>
-        <Icon type={isExpanded ? 'iconfont-keybase' : 'iconfont-down-arrow'} inheritColor={true} />
+        <Icon type={isExpanded ? 'iconfont-up-arrow' : 'iconfont-down-arrow'} inheritColor={true} />
```

Comment:
When is the up arrow here actually used? It looks like when I click on the down arrow, this Divider turns into the FloatingDivider below.

Reply:
If you have a lot of simple and big teams it'll show up. The floating divider shows if the big teams are below the fold, but if the big teams are above the fold then the regular divider shows (with the up arrow)

Reply:
![sim](https://user-images.githubusercontent.com/516435/29926989-f9cc4a18-8e32-11e7-9ab3-64b286501a0e.gif)